realm/realm-library/src/androidTest/java/io/realm/RealmListTests.java

```diff
@@ -962,58 +771,31 @@ public void removeAllFromRealm_outsideTransaction() {
     }
 
     @Test
-    public void removeAllFromRealm_listWithStandaloneObjectShouldThrow() {
```

Comment:
Yes, since we now split into `remove` and `delete` methods. This behaviour have now changed making the test no longer needed.

`deleteAllFromRealm` will throw UnsupportedOperation on the un-managed list.
`removeAll` doesn't care about managed/unmanaged, it just checks object equality.